test/new_device_startmode_test.rb

```diff
@@ -53,9 +58,11 @@
         expect(result).to be_eql expected_startmode
       end
 
-      it "results to ifplugd when running on laptop" do
+      it "results to ifplugd when running on laptop with wicked and not virtual device" do
```

Comment:
What exactly "not virtual device" means?

Reply:
Well, basically this results in ifplugd if none of the other conditions are true, not just because running on a laptop

https://github.com/yast/yast-network/blob/3d5ca9554e975d78ecadcda3a428361029530e52/test/new_device_startmode_test.rb#L79

Reply:
currently it means not a "bridge", "bond" or "vlan"